zuul-core/src/main/java/com/netflix/zuul/message/http/HttpQueryParams.java

```diff
@@ -270,6 +285,18 @@ public boolean equals(Object obj)
     @RunWith(MockitoJUnitRunner.class)
     public static class TestUnit
     {
+        Properties testProperties = new Properties();
+
+        private void setKeepOriginalEncoding(boolean keepOriginalEncoding) {
```

Comment:
nit: private methods should probably go at the bottom of the class to allow reading the test code from top to bottom.